schemas/core/outputs.json

```diff
@@ -96,6 +96,9 @@
         "output": {
           "type": "string"
         },
+        "fullRawLlmOutput": {
+          "type": "any"
```

Comment:
`any` isn't a valid type.  The way we've been handling this so far is by referencing this guy: https://github.com/guardrails-ai/interfaces/blob/main/schemas/core/any-type.json which is a simplified version of the meta schema